test/Microsoft.AspNet.Mvc.Razor.Test/RazorPageTest.cs

```diff
@@ -782,7 +782,7 @@ public void IsSectionDefined_ThrowsIfPreviousSectionWritersIsNotRegistered()
                     {
                         GetTagHelperOutput(
                             tagName:     "p",
-                            attributes:  new Dictionary<string, object>() { { "test", "testVal" } },
+                            attributes:  new TagHelperAttributes() { ["test"] = "testVal" },
```

Comment:
revert unnecessary churn.  you've got an `Add(string, object)` method available; no reason to switch to the long-form init syntax.  if you prefer this syntax and others agree, then do as a separate, broader PR.